src/Evolution/Executables/GrMhd/ValenciaDivClean/EvolveValenciaDivCleanAnalyticData.hpp

```diff
@@ -0,0 +1,177 @@
+  using ordered_list_of_primitive_recovery_schemes = tmpl::list<
+      grmhd::ValenciaDivClean::PrimitiveRecoverySchemes::NewmanHamlin,
+      grmhd::ValenciaDivClean::PrimitiveRecoverySchemes::PalenzuelaEtAl>;
+
+  using compute_rhs = tmpl::flatten<
+      tmpl::list<Actions::ComputeVolumeFluxes,
+                 dg::Actions::SendDataForFluxes<EvolutionMetavars>,
+                 Actions::ComputeVolumeSources, Actions::ComputeTimeDerivative,
+                 dg::Actions::ReceiveDataForFluxes<EvolutionMetavars>,
+                 tmpl::conditional_t<local_time_stepping, tmpl::list<>,
+                                     dg::Actions::ApplyFluxes>,
+                 Actions::RecordTimeStepperData>>;
+
+  using update_variables = tmpl::flatten<tmpl::list<
+      tmpl::conditional_t<local_time_stepping,
+                          dg::Actions::ApplyBoundaryFluxesLocalTimeStepping,
+                          tmpl::list<>>,
+      Actions::UpdateU, SlopeLimiters::Actions::SendData<EvolutionMetavars>,
+      SlopeLimiters::Actions::Limit<EvolutionMetavars>,
+      VariableFixing::Actions::FixVariables<VariableFixing::FixConservatives>,
+      Actions::UpdatePrimitives>>;
+
+  struct EvolvePhaseStart;
+  using component_list = tmpl::list<DgElementArray<
+      EvolutionMetavars, grmhd::ValenciaDivClean::Actions::Initialize<3>,
+      tmpl::flatten<tmpl::list<
+          VariableFixing::Actions::FixVariables<
+              VariableFixing::FixToAtmosphere<thermodynamic_dim>>,
+          Actions::UpdateConservatives,
+          SelfStart::self_start_procedure<compute_rhs, update_variables>,
+          Actions::Label<EvolvePhaseStart>, Actions::AdvanceTime,
+          VariableFixing::Actions::FixVariables<
+              VariableFixing::FixToAtmosphere<thermodynamic_dim>>,
+          Actions::UpdateConservatives, Actions::FinalTime,
+          tmpl::conditional_t<local_time_stepping,
+                              Actions::ChangeStepSize<step_choosers>,
+                              tmpl::list<>>,
+          compute_rhs, update_variables, Actions::Goto<EvolvePhaseStart>>>>>;
+
+  using const_global_cache_tag_list =
+      tmpl::list<analytic_data_tag,
+                 OptionTags::TypedTimeStepper<tmpl::conditional_t<
+                     local_time_stepping, LtsTimeStepper, TimeStepper>>,
+                 OptionTags::DampingParameter>;
+
+  using domain_creator_tag = OptionTags::DomainCreator<3, Frame::Inertial>;
+
+  static constexpr OptionString help{
+      "Evolve analytic data using the Valencia formulation of the GRMHD system "
+      "with divergence cleaning.\n\n"
+      "Analytic data: cylindrical blast wave\n"
+      "Slope limiter: Minmod family\n"
+      "Numerical flux: Local Lax-Friedrichs\n"
+      "Variable fixing: FixConservatives and FixToAtmosphere\n"
+      "Local time-stepping: none\n"
+      "Boundary conditions: only periodic are currently supported\n"};
+
+  enum class Phase { Initialization, Evolve, Exit };
```

Comment:
It‘ll presumably be just the current grmhd observer with everything that needs an analytic solution deleted, to start with. Out of scope for this PR I suppose, but will be necessary for @geoffrey to rebase the blast wave.

Reply:
Yes, I'm not sure what he is observing. I think often you don't actually want to observe all the variables anyway because it's a lot of data for things you might not care about.